README.md

```diff
@@ -161,6 +162,11 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [Zellij](https://github.com/zellij-org/zellij) - A terminal workspace with batteries included
 
+#### csv
```

Comment:
This section should be original command line tool to which xsv and csview are alternatives. If `csv` a command line tool?

Reply:
This is not the original command line tool, and I am not sure if it exists. So where should I put `csv`?

Reply:
If that's so, the two CLIs don't have an original CLI. So, I think they should not be added even though they are useful tools.

Reply:
I think they should be added to [awesome-rust](https://github.com/rust-unofficial/awesome-rust).

Reply:
Okay, then I close it this pr.

Reply:
Thank you for trying to contribute to this repository.